packages/fast-colors/src/colorlib/colorLAB.ts

```diff
@@ -0,0 +1,45 @@
+// tslint:disable:member-ordering
+import { roundToPrecisionSmall } from "./mathUtils";
+
+// CIELAB color space
+// https://en.wikipedia.org/wiki/CIELAB_color_space
+// This implementation uses the D65 constants for 2 degrees. That determines the constants used for the pure white point of the XYZ space of 0.95047, 1.0, 1.08883.
+// https://en.wikipedia.org/wiki/Illuminant_D65
+// These constants determine how the XYZ, LCH and LAB colors convert to/from RGB.
+export class ColorLAB {
+    public static readonly epsilon: number = 216 / 24389;
+    public static readonly kappa: number = 24389 / 27;

```

Comment:
Not a big deal since the impact would be small, but why not just have these as `0.00885645168` and `903.296296`?

Reply:
I ended up using the rational values based on what I read here: http://brucelindbloom.com/index.html (his site does something odd with navigation that makes html links not go to sub pages, go to Info -> A (dis)continuity study of the CIE L* Function).

Reply:
I see the section has "The CIE's decision to represent these constants as decimal values was an unnecessary approximation that introduces both function and slope discontinuities.", so looks good to me.